packages/consensus/src/validation/coinbase.cairo

```diff
@@ -160,7 +244,7 @@ mod tests {
 
 
     #[test]
-    fn test_validate_coinbase_with_multiple_input() {
+    fn _with_segwit_tx_with_multiple_input() {
```

Comment:
Why renaming?

Reply:
I think that was part of the incoming changes when i was rebasing

Reply:
At least that’s not the case for main branch right now, can you fix pls?